doc/examples/plot_restoration.py

```diff
@@ -4,14 +4,21 @@
 Image Deconvolution
 =====================
 
-In this example, we deconvolve a noisy version of an image using Wiener
-and unsupervised Wiener algorithms. This algorithms are based on
-linear models that can't restore sharp edge as much as non-linear
-methods (like TV restoration) but are much faster.
+In this example, we deconvolve a noisy version of an image using
+Richardson–Lucy, Wiener, and unsupervised Wiener deconvolution algorithms.
+This algorithms are based on linear models that can't restore sharp edge as
+much as non-linear methods (like TV restoration) but are much faster.
 
-Wiener filter
+Richardson–Lucy deconvolution
 -------------
-The inverse filter based on the PSF (Point Spread Function),
+The inverse filter based on the PSF (Point Spread Function).
+Regularisation is tuned through the number of iterations, which must
```

Comment:
I think regularization is used more than regularisation, as a quick google search confirmed (but it might be a British vs American thing).